treeherder/config/urls.py

```diff
@@ -2,23 +2,19 @@
 from django.conf.urls import (include,
                               url)
 from django.contrib import admin
-from django_browserid.admin import site as browserid_admin
 
 from treeherder.credentials.urls import urlpatterns as credentials_patterns
 from treeherder.embed import urls as embed_urls
 from treeherder.webapp.api import urls as api_urls
-from treeherder.webapp.views import LoginView
 
-browserid_admin.copy_registry(admin.site)
+admin.site.login_template = 'webapp/admin_login.html'
```

Comment:
Nice! I like that we can just get rid of the custom admin and replace with this template :-)
(I'm wondering if it wasn't possible to override the default template in the past, and that's why django-browserid did this)